README.md

```diff
@@ -20,7 +20,7 @@ Welcome! Here you will find everything you need to know to get started with and
 
 ---
 
-[**How-to**](#how-to) guides are **goal-oriented** recipes for accomplishing specific tasks. These are also good to browse to get an idea of how Ash Authentication works and what is possible with it.
+[**How-to**](#how-to) (**WIP**) guides are **goal-oriented** recipes for accomplishing specific tasks. These are also good to browse to get an idea of how Ash Authentication works and what is possible with it.
```

Comment:
If there are no how-to guides in AshAuthentication right now then we should probably drop this line item.

Reply:
I wasn't ready to be the one to make that decision. Since it *seemed* they had a purpose. Better to decide on that later? or do you want me to just remove it?

Reply:
Yeah, let's remove it until we have some :)